staging/BUILD

```diff
@@ -104,6 +104,7 @@ filegroup(
         "//staging/src/k8s.io/client-go/discovery:all-srcs",
         "//staging/src/k8s.io/client-go/dynamic:all-srcs",
         "//staging/src/k8s.io/client-go/examples/create-update-delete-deployment:all-srcs",
+        "//staging/src/k8s.io/client-go/examples/fake-client:all-srcs",
```

Comment:
@ixdy (please route to better person if not you)--Is there a reason why we need this BUILD file? If so, can we make it not change for every subdir added? It kind of defeats the purpose of having nested OWNERS files.

Reply:
The `package-srcs` glob only extends recursively to the next deepest package (i.e. `BUILD` file), so we have the `all-srcs` rule which includes that package.

We could create `BUILD` files in the root of all of the staging repos; then only those packages would be listed here, and subsequent changes underneath each repo would affect only that `BUILD` file.

This is why, for example, there is a `pkg/BUILD` file - there's no go sources in `pkg/`, but it prevents any directory changes in `pkg/` from percolating up to the root `OWNERS` file.

Reply:
(I'd be happy to do this change in a follow-up PR. I've got a different PR open right now affecting a bunch of `BUILD` files, and I'd like to avoid conflicting with myself.)